db/seeds.rb

```diff
@@ -6,7 +6,7 @@
 
 if user.nil?
   puts "Woah there, there aren't any users! Creating an user (#{email})."
-  user = User.create!(email:, full_name: "Stagey McStageface")
+  user = User.create!(email:, full_name: "Stagey McStageface", phone_number: "+18556254225")
```

Comment:
```suggestion
  user = User.create!(email:, full_name: "Stagey McStageface", phone_number: "+19064225632")
```
Just so we have a way of accessing messages if needed